### ISSUE TITLE
Repeated Bind Variables in STMT thrown assertion error

### ISSUE THREAD

**Issue by cwiese** [felipenoris/Oracle.jl#15]
when the stmt has the same :bind_var in more than one location in stmt .. this is thrown     

in get_bind_names ==>    @assert expected_num_bind_names == num_bind_names_ref[]

It is is a common pattern to have the same named bind var more that once in a SQL stmt ...

**Comment by cwiese** [comment]
perhaps this too should compare to "unique" count https://github.com/felipenoris/Oracle.jl/blob/df5410b355de28fcd7cbf8bf3f860fbf41ece2e8/src/stmt.jl#L80

**Comment by felipenoris** [comment]
Hi Chuck! Do you have a code example thst triggers the error?

**Comment by cwiese** [comment]
Hey Felipe !  How are things in Brazil?  

looks like it is this simple ..I need to use the same bind variable (this is a simplified SQL) 
```
 sql = "selecty * from (select col1, col2 from tab1 where :id union all select col1, col2 from tab2 where id = :id)"
conn = Oracle.Connection("user", "pass", "url")
 stmt = Oracle.Stmt(conn, sql)
```

**Comment by felipenoris** [comment]
Closed by 75873d6.
